tests/CMakeLists.txt

```diff
@@ -69,7 +73,3 @@ add_test(creategrib creategrib)
 add_executable(test_getgb2 test_getgb2.f90)
 target_link_libraries(test_getgb2 g2_d)
 add_test(test_getgb2 test_getgb2)
-
-add_executable(test_getdim test_getdim.f90)
-target_link_libraries(test_getdim g2_d)
-add_test(test_getdim test_getdim)
```

Comment:
Why are we removing this test?

Reply:
This should not be removed. Perhaps, I started from a old version of CMakelist. It will be corrected soon after merge with your latest commit.